src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,18 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+	public String getPrintableString() {
+		return getPrintableString(this.name, this.phone, this.email, this.address);
+	}
+	
+	//Printable is an interface ,name,phone,email,address implement it
+	private String getPrintableString(Printable... printables) {
+		String returnString = printables[0].getPrintableString();
```

Comment:
You could have used `StringJoiner` or `String.join` instead. Can consider StringBuilder as well